packages/compass-data-modeling/src/components/export-diagram-modal.tsx

```diff
@@ -73,34 +73,33 @@ const ExportDiagramModal = ({
       setOpen={onCloseClick}
       data-testid="export-diagram-modal"
     >
-      <ModalHeader
-        title="Export data model"
-        subtitle={
-          <div>
-            Export your data model as either an image or JSON file.
-            {nbsp}
-            <Link
-              href="https://www.mongodb.com/docs/manual/data-modeling//"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Learn more
-            </Link>
-          </div>
-        }
-      />
+      <ModalHeader title="Export data model" />
       <ModalBody className={modelBodyStyles}>
         <div className={contentContainerStyles}>
           <Label htmlFor="">Select file format:</Label>
           <RadioGroup className={contentContainerStyles} value={exportFormat}>
             <div className={radioItemStyles}>
-              <Icon glyph="Diagram2" />
+              <Icon glyph="Diagram" />
+              <Radio
+                checked={exportFormat === 'diagram'}
+                value="diagram"
+                aria-label="Diagram File"
+                onClick={() => onSelectFormat('diagram')}
+                size="small"
+                description="Importable into Compass so teammates can collaborate."
```

Comment:
That's a leafygreen thing, but this description not being clickable is such a weird design decision 😬

Reply:
agreed, i was thinking of keeping it outside of the radio component for this purposes, but to keep things cleaner, choose this way :)